EncoreTickets.SDK/Basket/Models/Seat.cs

```diff
@@ -4,9 +4,9 @@ public class Seat
     {
         public string aggregateReference { get; internal set; }
 
-        public string blockId { get; internal set; }
+        public string areaId { get; internal set; }
 
-        public string blockName { get; internal set; }
+        public string areaName { get; internal set; }
```

Comment:
Are you sure you need an internal modifier for model setters?

Reply:
It's currently used as a response model so I'm not sure why would we not make it readonly outside the SDK. Do you think we might need public access here?

Reply:
In the future, we can reuse models to add or update entities. I think this is an additional protection that is not very useful at the moment, but may be changed in the future for the purposes of unit testing and extensions of services